config/reference_container.yaml

```diff
@@ -0,0 +1,8 @@
+ 
+reference_container_folder: "/home/sven/Documents/reference_container/"
+reference_container_names:
```

Comment:
open question: is it possible to define an empty list in YAML? in other words: can this option be part of the template parameter config, but is read as empty list (= no names set) if not specified?

Reply:
The code right now checks if, through a config entry, a reference container should be used. If True there will be checks and appropriate and explicit error messages pop up if checks fail. If set to False an empty list (""), regardless if container_folder and/or container_names haven been set in a YAML, will be passed on, but that needs more testing if that works properly.